cmd/lakectl/cmd/doctor.go

```diff
@@ -47,46 +73,45 @@ var doctorCmd = &cobra.Command{
 
 		serverEndpoint := cfg.Values.Server.EndpointURL
 		if !strings.HasSuffix(serverEndpoint, api.BaseURL) {
-			Fmt("Suspicious URI format for server.endpoint_url: %v, doesn't have `%v` suffix.\n", serverEndpoint, api.BaseURL)
+			Fmt("Suspicious URI format for server.endpoint_url: %v, doesn't end with: `%v`.\n", serverEndpoint, api.BaseURL)
```

Comment:
Can we also handle this as a write to the template?  Otherwise we again have multiple places to control what the details look like.